app/api/chat/conversation/[slug]/message/route.ts

```
@@ -66,6 +71,15 @@ export const POST = withWidgetAuth<{ slug: string }>(async ({ request, context:
   }
 
   const userMessage = await createUserMessage(conversation.id, userEmail, content, attachmentData);
+  try {
+    await triggerEvent(
+      "conversations/auto-response.create",
+      { messageId: userMessage.id, tools },
+      { sleepSeconds: 5 * 60 },
+    );
+  } catch (err) {
+    console.error("Failed to enqueue auto-response", err);
```

ESLint

[error] Unexpected console statement.

(no-console)